default.nix

```
@@ -31,8 +31,8 @@ let
               # goreleaser (used for builds distributed via terraform registry) requires that CGO is disabled
               CGO_ENABLED = 0;
               ldflags = [ "-s" "-w" "-X main.version=${version}" "-X main.commit=${source.rev}" ];
```

remove source.rev here if it's not an accurate description oh what is build?
```suggestion
              ldflags = [ "-s" "-w" "-X main.version=${version}" ];
```